core/src/components/segment-view/segment-view.scss

```diff
@@ -0,0 +1,31 @@
+// Segment View
+// --------------------------------------------------
+
+:host {
+  display: flex;
+
+  height: 100%;
+
+  overflow-x: scroll;
+  scroll-snap-type: x mandatory;
+
+  /* Hide scrollbar in Firefox */
+  scrollbar-width: none;
+
+  /* Hide scrollbar in IE and Edge */
+  -ms-overflow-style: none;
+}
+
+/* Hide scrollbar in webkit */
+:host::-webkit-scrollbar {
+  display: none;
+}
+
+:host(.segment-view-disabled) {
+  scroll-snap-type: none;
+
+  opacity: 0.7;
```

Comment:
This number is made up — need to get with design to determine if this is wanted or not.

Reply:
I spoke with design about this and we decided to use the same value as segment uses when disabled.